### DISCUSSION TITLE
how to use --time-style=FORMAT with relative

### DISCUSSION THREAD

**Discussion by danielb2** [eza-community/eza#758] [Q&A, Howto, Help]
I'd like to show both relative and iso-long together. I can replicate iso-long easily with regular `+"%F %R"`, but how can I add relative?

**Comment by gierens** [comment]
The `--time-style relative` is a special eza mode while the patterns like `--time-style '+%F %R'` are handed to the `chrono` crate for expansion. chronos format specifiers are listed here: https://docs.rs/chrono/latest/chrono/format/strftime/index.html . It does not seem to support relative formatting. So this is not possible at the moment.

Theoretically we could add another mode for this, but we would need a good design first. Just a 'full-iso-and-relative` seems a little niche to me.

**Reply by Diti** [reply]
Personally, I would use `--time-style %F %R<newline><RELATIVE> %R` to show old files in ISO format and recent files in relative format.
